server/audit_math/bravo.py

```diff
@@ -374,7 +382,7 @@ def get_sample_size(
             best_loser = loser
 
     # If we're in a single-candidate race, set sample to 0
-    if margin["losers"] == 0:
+    if not margin["losers"]:
```

Comment:
why this change?

Reply:
`margin["losers"]` contains an object, it's not a number.